### DISCUSSION TITLE
Training is slower in Baal 1.5.2

### DISCUSSION THREAD

**Discussion by rajankita** [baal-org/baal#207] [Q&A]
I was working with baal 1.5.1 earlier, and on my dataset and model configuration, it took me around 90 seconds to train a single epoch. I upgraded to 1.5.2 in order to use the unpatch_module, and now the training is super slow. I haven't changed anything with my code. Is anyone else facing similar issues?

**Comment by Dref360** [comment]
Hmm strange, would you mind sharing your script? 

If you could give me the output of `pip freeze` that would help too.

Feel free to reach out to me on Slack if you do not want to share your training script publicly.

I did update dependencies in https://github.com/ElementAI/baal/pull/184/files, so that might be the cause of it.
